test/micro/org/openjdk/bench/java/lang/VectorBitConversion.java

```diff
@@ -0,0 +1,125 @@
+package org.openjdk.bench.java.lang;
```

Comment:
I think this benchmark belongs with the other vectorization benchmarks under
`test/micro/org/openjdk/bench/vm/compiler/Vector*`

Reply:
It's not really a language feature, more for vm/compiler